spec/system/signup/shared_context.rb

```diff
@@ -1,7 +1,7 @@
 RSpec.shared_context "signup", shared_context: :metadata do
   before(:all) do
-    unless ENV["GOOGLE_API_CREDENTIALS"] && ENV["RADIUS_KEY"] && ENV["RADIUS_IPS"]
-      abort "\e[31mMust define GOOGLE_API_CREDENTIALS, RADIUS_KEY, and RADIUS_IPS\e[0m"
+    unless ENV["GOOGLE_API_CREDENTIALS"] && ENV["RADIUS_KEY"] && ENV["RADIUS_IPS"] && ENV["SUBDOMAIN"]
+      abort "\e[31mMust define GOOGLE_API_CREDENTIALS, RADIUS_KEY, SUBDOMAIN and RADIUS_IPS\e[0m"
     end
```

Comment:
`SUBDOMAIN` between the two `RADIUS_` vars?

Reply:
I've swapped these round now so it's more readable.